src/main/java/org/chocosolver/solver/variables/impl/SetVarImpl.java

```diff
@@ -130,6 +134,29 @@ public ISet getLB() {
 	public ISet getUB() {
 		return ubReadOnly;
 	}
+	
+	@Override
+	public IntVar getCard() {
+		if(cardinality==null){
+			int ubc =  ub.getSize(), lbc = lb.getSize();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://www.codacy.com/app/cprudhom/choco-solver/file/2534693622/issues/source?bid=3218989&fileBranchId=3284892#l141)